Formula/irrlicht.rb

```diff
@@ -7,6 +7,7 @@ class Irrlicht < Formula
 
   bottle do
     cellar :any_skip_relocation
+    sha256 "88ef6a24a237c71b1342f4cd6a8dbf6d797d2460b1020cdf7f49b7ef0d2a24a2" => :mojave
```

Comment:
Can you please remove this line? A bottle will be automatically built when we merge it. Thanks!